docker-compose.dev.yml

```diff
@@ -4,7 +4,7 @@ services:
         volumes:
```

Comment:
Let's make sure that these docker compose files get updated to run everything at once in development.